src/NodeJS/Server.php

```diff
@@ -254,6 +262,52 @@ public function getConnection()
     }
 
     /**
+     * Return the options.
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return $this->options;
+    }
+
+    /**
+     * Set options array.
+     */
+    public function setOptions($options)
+    {
+        // pass these to the setOption() method which will validate.
+        foreach ($options as $key => $value) {
+            $this->setOption($key, $value);
+        }
+    }
+
+    /**
+     * Set a single option.
+     */
+    public function setOption($option, $value)
+    {
+        $valid_options = array(
+            'features',
+            'headers',
+            'waitDuration',
+            'proxy',
+            'referrer',
+            'silent',
+            'site',
+            'strictSSL',
+            'userAgent',
+            'language',
+            'runScripts',
+            'localAddress',
+        );
+
+        if (in_array($option, $valid_options)) {
```

Comment:
That's not normal case, when non-supported option is given, so better to throw an exception immediately.